java/client.cfg

```diff
@@ -174,7 +174,7 @@ MAX_CURSOR=100
 INITIALIZE_NEW=1
 ISOLATION_LEVEL=CR
 XBASE_TINT=1
-DBMS=sqlserver
+DBMS=sqlite
```

Comment:
This change was intended?

Reply:
Yes, unit test must run using sqlite instead of sqlserver